Sources/MapboxMapsSwiftUI/MapboxView.swift

```diff
@@ -0,0 +1,87 @@
+@_exported import MapboxMaps
+import SwiftUI
+
+public extension CameraState {
+    /// Initializes CameraState with center and zoom.
+    init(center: CLLocationCoordinate2D, zoom: CGFloat) {
+        self.init(center: center, padding: .zero, zoom: zoom, bearing: 0, pitch: 0)
```

Comment:
You must be talking about the `CameraOptions` which has the nullable fields and represents "request" for change some properties.

The `CameraState` is different - it represents actual values of the camera and thats what we want for SwiftUI, since `@State` represents the source of truth for any value.

 If the user wants to update only several properties, they can just modify them: 
```swift
Button("Change Camera") {
  camera.bearing = 180
  camera.pitch = 20
}
```